components/layout/menu.hbs

```diff
@@ -1,316 +1,293 @@
-                href="https://aquent.co.uk"
-                title="UK"
+                class="anchor--primary"
+              ><span class="p--body-small">Recruitment Marketplace</span></a></li>
+            <li class="sub-heading" role="none"><span
+                class="tertiary-info"
+              >Tools</span></li>
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="Persona"
                 role="menuitem"
-              >UK</a></li>
+                class="anchor--primary"
+              ><span class="p--body-small">Persona</span></a></li>
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="Book"
+                role="menuitem"
+                class="anchor--primary"
+              ><span class="p--body-small">Book</span></a></li>
+          </ul>
+        </div>
+      </details>
+      <details><summary class="tertiary-info--large">
+          <span class="menuitem">
+            <i class="arrow--before"></i>Who we are<i class="arrow--after"></i>
+          </span>
+        </summary>
+        <div class="dropdown-menu section--bg-white" role="menubar">
+          <ul class="dropdown-menu-main" role="menubar">
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="About"
+                role="menuitem"
+                class="anchor--primary"
+              ><span class="p--body-small">About</span></a></li>
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="Our People"
+                role="menuitem"
+                class="anchor--primary"
+              ><span class="p--body-small">Our People</span></a></li>
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="Client Stories"
+                role="menuitem"
+                class="anchor--primary"
+              ><span class="p--body-small">Client Stories</span></a></li>
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="Communities"
+                role="menuitem"
+                class="anchor--primary"
+              ><span class="p--body-small">Communities</span></a></li>
           </ul>
-        </li>
-      </ul>
+        </div>
+      </details>
+      <a
+        href="#"
+        title="Contact Us"
+        role="menuitem"
+        class="button--small-primary"
+      >Contact Us</a>
+    </div>
+    <div
+      class="menu-toggle inactive"
+      id="menu-toggle"
+      data-aria-label-open="Open menu"
+      data-aria-label-close="Close Menu"
+      aria-label="Open menu"
+    >
+      <a class="" href="#navbar" id="open-menu">
+        <span class="show-for-sr">Open Menu</span>
+      </a>
+      <a class="" href="#" id="close-menu">
+        <span class="show-for-sr">Close Menu</span>
+      </a>
     </div>
   </div>
 </nav>
```

Comment:
Can we get some HTML comments on these identifying which tags they are specifically closing so it's easy at a glance?